app/routes/app/ecosystem/metrics.tsx

```diff
@@ -21,9 +21,9 @@ export default function Metrics() {
           <p className="font-semibold text-md">Activity</p>
           <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-5 mx-auto items-center">
             <StatsCard figure={87} title="Marketplaces" />
-            <StatsCard figure={2000} title="Challenges" />
-            <StatsCard figure={50000} title="Submissions" />
-            <StatsCard figure={175000} title="Reviews" />
+            <StatsCard figure={"2,000"} title="Challenges" />
+            <StatsCard figure={"50,000"} title="Submissions" />
+            <StatsCard figure={"175,000"} title="Reviews" />
```

Comment:
Better way to get the proper formatting is to use numbers like before but in StatsCard you render it with `figure.toLocaleString()`